webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -346,7 +347,7 @@ def test_quick_search(self):
         # Test with a signature.
         response = self.client.get(url, {"query": "moz"})
         assert response.status_code == 302
-        target = reverse("supersearch:search") + "?signature=%7Emoz"
+        target = reverse("supersearch:search") + "?signature=~moz"
```

Comment:
`%7E` is `~`. I think something changed between 3.6 and 3.7 such that the `response["location"]` isn't urlencoding the `~` anymore. I didn't look into it.